src/app/shared/components/workshop-card/workshop-card.component.scss

```diff
@@ -26,3 +26,65 @@
     color: #C72A21;
   }
 }
+
+.horizontal-card {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  overflow: hidden;
+  .card-img {
+    display: flex;
+    width: 100%;
+    min-width: 150px;
+    object-fit: cover;
+    object-position: bottom;
+  }
+  .mat-card-image{
+    width: 100%;
+    margin: 0;
+    margin-bottom: 0 !important;
```

Comment:
you can avoid using important by adding additional selector to increase priority